pkg/transfers/router.go

```diff
@@ -226,7 +226,11 @@ func CreateTransfer(
 
 			var companyID string
 			orgConfig, err := orgRepo.GetConfig(responder.OrganizationID)
-			if err == nil && orgConfig != nil {
+			if err != nil {
+				responder.Problem(fmt.Errorf("getting org config: error getting config: %v", err))
+				return
+			}
+			if orgConfig != nil {
```

Comment:
While we're in the code can we remove that old TODO(adam) of mine? This change to read from the db resolves it.